app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def update_post_counter(number)
+    author.update_attribute 'posts_counter', number
+  end
+
+  def recent_five_comments
+    comments.order(updated_at: :desc).last(5)
+  end
```

Comment:
[OPTIONAL]

- [ ] I would recommend adding the `after_save` callback. It will call the `#update_post_counter` method every time a new post has been saved to database.

```ruby
# app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter(number)
    author.update_attribute 'posts_counter', number
  end
```

Reply:
Thank you so much for your valuable feedback!